src/main/java/com/algorand/sdkutils/listeners/JavaGenerator.java

```diff
@@ -343,6 +343,7 @@ public void generateEnumClasses (String rootPath, String pkg)  {
         BufferedWriter bw = Tools.getFileWriter("Enums", rootPath);
         try {
             bw.append("package " + pkg + ";\n\n");
```

Comment:
@winder Unrelated to the line:  To vet the generated result, can you supply the corresponding java-algorand-sdk PR?

I'd like to review it prior to approving here.

Reply:
Yes, see the PR here: https://github.com/algorand/java-algorand-sdk/pull/351

Reply:
@winder Thanks for opening the PR - resolving.